JWBestPracticeApps/NativeControls/ViewController.swift

```diff
@@ -0,0 +1,315 @@
+//
+//  ViewController.swift
+//  NativeControls
+//
+//  Created by David Almaguer on 8/21/19.
+//  Copyright © 2019 Karim Mourra. All rights reserved.
+//
+
+import UIKit
+
+let LIVEString = "LIVE"
+let DefaultTimeString = "00:00"
+
+class ViewController: JWBasicVideoViewController {
+    
+    @IBOutlet weak var controlsView: UIView!
+    @IBOutlet weak var replayButton: UIButton!
+    @IBOutlet weak var playbackButton: UIButton!
+    @IBOutlet weak var forwardButton: UIButton!
+    @IBOutlet weak var audioButton: UIButton!
+    @IBOutlet weak var fullscreenButton: UIButton!
+    @IBOutlet weak var timeLabel: UILabel!
+    @IBOutlet weak var timeSlider: UISlider!
+    
+    let indicatorView = UIActivityIndicatorView(style: .whiteLarge)
+    
+    // Array with tuples (Title, URL)
+    var feed: [(String, String)] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Verify instance of JWPlayerController
+        if let player = self.player {
+            // Hide JWPlayer's controls to use your own native contorls
+            player.config.controls = false
+            
+            // Add an observer for JWPlayerStateChangedNotification
+            NotificationCenter.default.addObserver(self, selector: #selector(stateChangedNotification(_:)), name: NSNotification.Name(rawValue: JWPlayerStateChangedNotification), object: nil)
+        }
+        
+        // Setup indicator view
+        indicatorView.color = UIColor(red: 234/255, green: 52/255, blue: 76/255, alpha: 1)
+        player.view.addSubview(indicatorView)
```

Comment:
This line and the line below are giving me compiling errors because player.view is a UIView?, and thus requiring optional chain to access addSubview(). Can you verify?